release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2995,6 +2995,57 @@ This section will continue to be updated over time to provide notes on enhanceme
++
+Workaround: The node will start running when the Podman version in the base image does not support Sigstore. Use the default `policy.json` file that does not have Sigstore verification if the base image is 4.11 or earlier. (link:https://issues.redhat.com/browse/OCPBUGS-48296[*OCPBUGS-48296*])
+
+* There is currently a known issue where the data image is still present after deleting a related bare-metal host.
++
+Workaround: Delete the data image if it exists after the bare-metal host has been deleted. (link:https://issues.redhat.com/browse/OCPBUGS-45250[*OCPBUGS-45250*])
+
+[id="ocp-4-18-4-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, you could not use catalog or bundle images that contained files with restricted extended attributes. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-52173[*OCPBUGS-52173*])
+
+* Previously, you could not use the `registryOverride` option to override catalog Operator images. With this release, the logic for the control plane Operator is updated, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-51375[*OCPBUGS-51375*])
+
+* Previously, the Installer failed to retrieve Google Cloud Platform (GCP) tags over an unstable network or when it could not reach the GCP server. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-51211[*OCPBUGS-51211*])
+
+* Previously, if you had permission to view nodes but not Certificate Signing Requests (CSR), you could not access the *Nodes list* page. With this release, permissions to view CSRs are no longer required to access the *Nodes list* page. (link:https://issues.redhat.com/browse/OCPBUGS-51149[*OCPBUGS-51149*])
+
+* Previously, the *Observe* section on the web console did not show items contributed from plugins unless certain flags related to monitoring were set. However, these flags prevented other plugins, such as logging, distributed tracing, network observability, and so on, from adding items to the *Observe* section. With this release, the monitoring flags are removed so that other plugins can add items to the *Observe* section. (link:https://issues.redhat.com/browse/OCPBUGS-51086[*OCPBUGS-51086*])
+
+* Previously, if `kubevirt` and `graphImage` images were not retrieved during the collection phase of `oc-mirror`, the run would succeed even if the images were missing. With this release, the `oc-mirror` run fails as expected if the images are not found. (link:https://issues.redhat.com/browse/OCPBUGS-50981[*OCPBUGS-50981*])
+
+* Previously, an issue prevented the configuration of multiple subnets in the failure domain of a Nutanix cluster during installation. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-49885[*OCPBUGS-49885*])
```

Comment:
Looks like this is missing the corresponding update to the KI entry that this bug had at GA. See lines 2870-2871 in https://github.com/openshift/openshift-docs/pull/89802/files